CONTRIBUTING.md

```diff
@@ -146,7 +142,7 @@ First, here's what my workflow looks like (which is quite similar to this
     ```shell
     git checkout master
     git fetch upstream
-    git rebase -i upstream master
+    git merge -i upstream master
```

Comment:
'-i' flag should be removed? Also add slash: "upstream/master".

Reply:
Done